pkg/build/api/v1/types.go

```diff
@@ -371,6 +375,13 @@ type SourceBuildStrategy struct {
 	ForcePull bool `json:"forcePull,omitempty" description:"forces the source build to pull the image if true"`
 }
 
+// BuildPostCommitSpec holds a build post commit hook specification.
+type BuildPostCommitSpec struct {
+	// RunBash holds a bash script to be executed in a container running the
+	// build output image.
+	RunBash string `json:"runBash,omitempty" description:"bash script to be executed in a container running the build output image"`
```

Comment:
if we bypass the entrypoint, this will not work for any of our s2i images without either
1) always prefixing bash -c (ew)
2) the user being forced to prefix bash -c manually (ew)

because the s2i images require various scl enable commands be run in order for "rake test" to function, and getting those commands run requires either a bash environment be setup.